src/GitHub.VisualStudio/Services/UsageService.cs

```diff
@@ -79,9 +113,9 @@ public IDisposable StartTimer(Func<Task> callback, TimeSpan dueTime, TimeSpan pe
                 var json = SimpleJson.SerializeObject(data);
```

Comment:
Note that this is going to serialize the fields with PascalCasing. I guess we need to write a `CamelCaseSerializerStrategy` if we want to serialize it using camelCase.

Reply:
It doesn't matter much here, the method that post the data serializes the model into camel case.